src/namespaces.jl

```diff
@@ -11,23 +11,32 @@ cached_namespaces = Dict{String, Module}()
 
 """
 Import an R package as a julia module.
+Sanitizes the imported symbols by default, because otherwise certain symbols cannot be used.
+Eg: PerformanceAnalytics::charts.Bar in R becomes PerformanceAnalytics.charts_Bar in Julia
```

Comment:
Needs backticks around code. Also an empty line after the short summary would be nice, and "Eg" should be "For example" or "E.g.".

Reply:
Changed and pushed. Thanks for the review.